src/libraries/Microsoft.PowerFx.Core/IR/IRContext.cs

```diff
@@ -25,5 +26,20 @@ public static IRContext NotInSource(FormulaType resultType)
         {
             return new IRContext(null, resultType);
         }
+
+        public override bool Equals(object obj)
```

Comment:
>public override bool Equals(object obj) [](http://example.com/codeflow?start=8&length=39)

This may be problematic - why do we need it?
It would be safer to just compare types.

Reply:
removed.